CHANGELOG.md

```diff
@@ -337,7 +337,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Test explorer will always show in the Activity Bar after the extension is activated. [#271](https://github.com/Microsoft/vscode-java-test/issues/271)
 
 ### Fixed
```

Comment:
In changelog, you could mention it "Rename the extension name to "Test Runner for Java"".

Reply:
Do you mean keep the previous change log calling Java Test Runner?